src/reducers/userSlice.js

```diff
@@ -0,0 +1,79 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import * as auth from 'auth-provider';
+
+const Status = {
+  idle: 'idle',
+  pending: 'pending',
+  resolved: 'resolved',
+  rejected: 'rejectedUnauthenticated',
```

Comment:
Suggestion: Using uppercase for constants `IDLE: 'IDLE', PENDING: 'PENDING',...`